src/api/channels.coffee

```diff
@@ -225,3 +228,53 @@ exports.removeChannel = (channelId) ->
   catch err
     # Error! So inform the user
     utils.logAndSetResponse this, 500, "Could not remove channel by id: #{id} via the API: #{e}", 'error'
+
+###
+# Manually Triggers Polling Channel
+###
+exports.triggerChannel = (channelId) ->
+
+  # Test if the user is authorised
+  # if authorisation.inGroup('admin', this.authenticated) is false
+  #   utils.logAndSetResponse this, 403, "User #{this.authenticated.email} is not an admin, API access to removeChannel denied.", 'info'
+  #   return
```

Comment:
Oh, I see you are doing this check below. It should be able to be done here rather similar to the other endpoints code.